bridges/otelzerolog/hook_test.go

```diff
@@ -103,3 +108,81 @@ func TestNewHook(t *testing.T) {
 		})
 	}
 }
+
+var (
+	testMessage = "log message"
+	loggerName  = "name"
+	testKey     = "key"
+	testValue   = "value"
+	testEntry   = zerolog.InfoLevel
+)
+
+func TestHookRun(t *testing.T) {
+	rec := logtest.NewRecorder()
+	hook := NewHook(loggerName, WithLoggerProvider(rec))
+
+	logger := zerolog.New(os.Stderr).Hook(hook)
+
+	t.Run("Run", func(t *testing.T) {
+		// Create an event and run the hook
+		event := logger.Info().Str(testKey, testValue)
+		hook.Run(event, testEntry, testMessage)
+
+		// Check the results
+		got := rec.Result()[0].Records[0]
```

Comment:
If for some reason, there are no results, you'll get a non-useful crash here.

```suggestion
		require.Len(t, rec.Result(), 1)
		require.Len(t, rec.Result()[0].Records, 1)
		got := rec.Result()[0].Records[0]
```